<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KCookiesManagementUI</class>
 <widget class="QWidget" name="KCookiesManagementUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>443</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="KTreeWidgetSearchLine" name="searchLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="clickMessage">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="5">
    <widget class="QTreeWidget" name="cookiesTreeWidget">
     <property name="toolTip">
      <string>Search interactively for domains and hosts</string>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>Site</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cookie Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>D&amp;elete</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="deleteAllButton">
     <property name="text">
      <string>Delete A&amp;ll</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="configPolicyButton">
     <property name="text">
      <string>Configure &amp;Policy...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="reloadButton">
     <property name="text">
      <string>&amp;Reload List</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>18</width>
       <height>114</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="detailsGroupBox">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>nameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="nameLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="valueLabel">
        <property name="text">
         <string>Value:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>valueLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="valueLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="domainLabel">
        <property name="text">
         <string>Domain:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>domainLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="domainLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="patchLabel">
        <property name="text">
         <string>Path:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>pathLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KLineEdit" name="pathLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="expiresLabel">
        <property name="text">
         <string>Expires:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>expiresLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KLineEdit" name="expiresLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="secureLabel">
        <property name="text">
         <string>Secure:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>secureLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="KLineEdit" name="secureLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
